spec/request/vet360/address_request_spec.rb

```diff
@@ -151,5 +156,63 @@
         expect(errors_for(response)).to include "address-pou - can't be blank"
       end
     end
+
+    context 'when effective_end_date is included' do
+      let(:address) do
+        build(:vet360_address,
+              vet360_id: user.vet360_id,
+              effective_end_date: Time.now.utc.iso8601)
+      end
+
+      before do
+        allow_any_instance_of(User).to receive(:icn).and_return('1234')
+        address.id = 42
```

Comment:
Without knowing any better, `42` feels like a magic number case.  Is this for alignment with a cassette?  If yes, perhaps create a method called something like `set_id_to_cassettes_value`, or something like that, that makes transparent why `42` is being used, and also eliminates extra inline commenting in order to do so.